ocrd_froc/processor.py

```diff
@@ -91,32 +90,32 @@ def _process_segment(self, segment, image):
                 if classification_result != '':
                     classification_result += ', '
                 classification_result += '%s:%d' % (typegroup, score)
-            
-            if output_font :
+
+            if output_font:
                 textStyle.set_fontFamily(classification_result)
-        
-        
-        if method == 'COCR' :
+
+
+        if ocr_method == 'COCR':
             fast_cocr = self.parameter['fast_cocr']
-            transcription, score = self.froc.run(image, 
-                                          method=method, 
+            transcription, score = self.froc.run(image,
+                                          method=ocr_method,
                                           fast_cocr=fast_cocr)
-        elif method == 'SelOCR' :
-            transcription, score = self.froc.run(image, 
-                                          method=method, 
+        elif ocr_method == 'SelOCR':
+            transcription, score = self.froc.run(image,
+                                          method=ocr_method,
                                           classification_result=result)
-        else :
+        else:
             fast_cocr = self.parameter['fast_cocr']
             adaptive_treshold = self.parameter['adaptive_treshold']
-            transcription, score = self.froc.run(image, 
-                                          method=method, 
-                                          classification_result=result, 
-                                          fast_cocr=fast_cocr, 
+            transcription, score = self.froc.run(image,
+                                          method=ocr_method,
+                                          classification_result=result,
+                                          fast_cocr=fast_cocr,
                                           adaptive_treshold=adaptive_treshold)
         segment.set_TextEquiv([TextEquivType(Unicode=transcription, conf=score)])
```

Comment:
We should have a way to react to existing text here:

- merely `add_TextEquiv` instead of replacing the list
- add a param `overwrite_text` which controls whether existing results should be removed